web/gatsby-config.js

```diff
@@ -8,6 +8,8 @@ const token = process.env.SANITY_READ_TOKEN;
 
 const isProd = process.env.NODE_ENV === 'production';
 
+console.log('this is config', clientConfig);
```

Comment:
@jamie-endeavour just this line to remove 👍

Reply:
Ta